observability/WFLY-20296-add-support-for-more-opentelemetry-config-options.adoc

```diff
@@ -0,0 +1,140 @@
+** The shape of the management model need not share the nested nature unless that is a more natural structure in the Java code
+** Shared properties (e.g., the OpenTelemetry Collector endpoint) should be exposed on the root element.
+*** Signal-specific overrides need not be supported
+* All new properties added will have, as defaults, values described in the relevant OpenTelemetry documentation.
+* Support should not depend on, e.g., MicroProfile Telemetry or MicroProfile Config
+* Any configured properties should still be able to be overridden via the existing MicroProfile support
+* While it is acceptable to relate WildFly management model attributes to their respective system property in the OpenTelemetry documentation to help explain their intent, the system property is an implementation detail that should not necessarily be leaked in the model to avoid as much coupling as is possible.
+
+=== Changed requirements
+
+N/A
+
+=== Non-Requirements
+
+While unifying observability configuration (i.e., that of `wildfly-opentelemetry` and `wildfly-micrometer`) has been requested and might be of interest to users and administrators, that is a non-goal for this RFE.
+
+=== Future Work
+
+N/A
+
+== Backwards Compatibility
+
+This change will, of necessity, change the management model of the subsystem. Despite this change, existing configurations should continue to work unchanged. Furthermore, existing configurations should be automatically migrated to the new model/schema once a relevant change has made to the configuration.
+
+=== Default Configuration
+
+No default values will be changed.
+
+=== Importing Existing Configuration
+
+Existing configurations should be supported unchanged.
+
+=== Deployments
+
+Deployments should see no change in behavior.
+
+=== Interoperability
+
+No interoperability concerns are foreseen.
+
+== Implementation Plan
+
+N/A
+
+== Admin Clients
+
+N/A
+
+== Security Considerations
+
+In general, this should have no impact on security. However, one of the proposed changes includes adding support for configuring TLS certificates for communicating with the OpenTelemetry Collector. The use of these TLS certificates will allow the server to both secure communications with the Collector, and verify that the collector is, indeed, the correct receiver for the signals. This will have a positive impact on the data security for the subsystem and user applications.
+
+[[test_plan]]
+== Test Plan
+
+The existing tests under `testsuite/integration/microprofile` will be extended to cover the relevant newly added properties. Not all properties (such as those affecting publish rates or batch sizes) need extensive integration testing. Others, though, such as the TLS properties, will need thorough testing, both positive and negative.
+
+== Community Documentation
+
+The existing documentation in the Administration Guide will be augmented to cover the new options
```

Comment:
Please be sure to require coverage of the relationships between new attributes, system props and MP config options, particularly the latter as the intent above is those will be supported.